util/Migrator/MigratorConstants.cs

```diff
@@ -4,5 +4,5 @@ public static class MigratorConstants
 {
     public const string SqlTableJournalName = "Migration";
     public const string DefaultMigrationsFolderName = "DbScripts";
-    public const string TransitionMigrationsFolderName = "DbScripts_data_migration";
+    public const string TransitionMigrationsFolderName = "DbScripts_transition";
```

Comment:
ℹ️ Also caught this. We haven't had any, but the self-host method that uses this would have not worked.

What about finalization migrations in self-host? The [docs](https://contributing.bitwarden.com/contributing/database-migrations/edd#offline-environments) say those don't run offline, but this is an issue right?

Reply:
They would be run on the next release when they are prompted from finalization to regular migrations?